ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -219,21 +234,40 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                 </ZeroState>
             );
         }
-        this.state.workflows.sort(compareWorkflows);
 
         return (
             <>
-                <div className='row'>
-                    <div className='columns small-12 xxlarge-12'>
-                        {this.state.workflows.map(workflow => (
-                            <div key={workflow.metadata.name}>
-                                <Link to={uiUrl(`workflows/${workflow.metadata.namespace}/${workflow.metadata.name}`)}>
-                                    <WorkflowListItem workflow={workflow} archived={false} />
-                                </Link>
-                            </div>
-                        ))}
+                <div className='argo-table-list'>
+                    <div className='row argo-table-list__head'>
+                        <div className='columns small-1' />
+                        <div className='columns small-4'>NAME</div>
+                        <div className='columns small-3'>NAMESPACE</div>
+                        <div className='columns small-2'>STARTED</div>
+                        <div className='columns small-2'>DURATION</div>
```

Comment:
Minor: Columns should preferably sum to 12

Reply:
1+4+3+2+2=12?